lib/init/features/project.js

```diff
@@ -57,7 +57,7 @@ function _getProject(options) {
     if (choices.length >= 25) {
       utils.logBullet(
         "Don't want to scroll through all your projects? If you know your project ID, " +
-          "you can initialize it directly using 'firebase --project=$PROJECT_ID init.\n"
+          "you can initialize it directly using 'firebase init --project=$PROJECT_ID.\n"
```

Comment:
Sorry, getting super pedantic here, but for consistency let's do:

    "you can initialize it directly using " + clc.bold("firebase init --project <project_id>") + ".\n";